src/App.jsx

```diff
@@ -29,8 +54,32 @@ const App = () => {
   }, [todoList, isLoading]);
 
 
-  const addTodo = (newTodo) => {
-    setTodoList([...todoList, newTodo]);
+  const addTodo = async (newTodo) => {
+    // setTodoList([...todoList, newTodo]);
+    console.log(newTodo);
+    const data = {
+      fields: {
+        title: newTodo.title,
+      },
+    };
+    const options = {
+      method: "POST",
+      headers: headers,
+      body: JSON.stringify(data),
+    };
+    try {
+      const response = await fetch(url, options);
+      if (!response.ok) {
+        throw new Error(`Error: ${response.status}`);
+      }
+      const updatedData = await response.json();
+      console.log(updatedData);
+      newTodo.id = updatedData.id;
+      setTodoList([...todoList, newTodo]);
```

Comment:
After setting the data make sure you gall the fetch data function here so that your list on the UI has same values with that in the database/airtable

Reply:
Sorry, I do not understand it. I added one new todo item to the airtable, got the inserted record id, and after that added this new todo item with a new id (gotten from airtable) into todoList. So, I am sure I have the same new todo item in the database/airtable and on the UI. 
Our list on the UI will differ from the list in the database in case we remove some items from the list (removeTodo function) because we do not synchronize this function with the database)

Reply:
it's a good practice to always get the freshest data from the db but I will let this pass

Reply:
Done. Not sure I did it correctly.